src/components/SocialMediaButtons/DiscordButton/DiscordButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import DiscordIcon from "./discord.svg";
+
```

Comment:
Can you can compound these social media buttons into one new component called SocialMediaButton or something to that effect that takes in a link, a logo and an alt? That way we only have one file/component to worry about.

Reply:
I've gone ahead and created the SocialMediaButton component, but instead of taking those 3 things separately, an Enum for each of these platforms dictates those 3 things bundled together.